.nvmrc

```diff
@@ -0,0 +1 @@
+v4.4.7
```

Comment:
This file is not required by this project and is specific to the users choice of node version manager. Instead could you add it the the .gitignore so that anyone add their own. with it being tracked by this project.